app/models/katello/glue/provider.rb

```diff
@@ -74,9 +74,13 @@ def owner_upstream_update(upstream, _options)
                                                       upstream['idCert']['key'], ca_file, params)
       end
 
+      def upstream_api_url(upstream)
+        upstream['apiUrl'] || ENV['UPSTREAM_REDHAT_RHSM_API_URL'] ||  API_URL
```

Comment:
In `upstream_job.rb` the variable is called `REDHAT_RHSM_API_URL`; here it's `UPSTREAM_REDHAT_RHSM_API_URL`. Is that intended?

Reply:
Moved it to the same value. No need to keep it separate.